src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -242,10 +242,10 @@ private void extractType() {
         this.extractTitle();
         this.extractType();
 
-        if (bibEntry.getType() == null) {
-            bibEntry.setType(BibEntry.DEFAULT_TYPE);
+        // we pass a new BibEntry in the constructor which is never empty as it already consists of "@misc"
+        if (bibEntry.equals(new BibEntry())) {
```

Comment:
On second thought, adding a `isEmpty` method in BibEntry might lead to more readable code.

Reply:
Right, your hint gave me the idea to check if the entry has any fields